src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
the reset file should be imported first, so it wont affect the cascade of other style statements.